.backportrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prTitle": "{{sourcePullRequest.title}} [backport: {{targetBranch}}]",
+  "targetPRLabels": ["backport"]
```

Comment:
After testing for a while, I found it difficult to understand backported PRs without a label, so I've added one.

- PRs that we want to backport: `backport-to-release/v0.50`
- PRs automatically backported: `backport`

CC: @simar7

Reply:
> PRs automatically backported: backport

What release would it backport to? Would it automatically pickup the patch or minor depending on (fix/chore/etc.) and (feat)?

Reply:
I've updated my comment. It's simple. If we label "backport-to-release/v0.52" to a PR, the tool "backport" will cherry-pick the commit from the PR in the main branch and create a new PR with label "backport" aginst the branch "release/v0.52". It's backporting from "main" to "release/v0.52".